docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2236,3 +2236,38 @@ defaultRoles()
 
 Type: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
 
+## getServerPort {#getserverport}
+
+Returns the number of the server port. When the port is not used by the server, throw an exception.
+
+**Syntax**
+
+``` sql
+getServerPort(port_name)
+```
+
+**Arguments**
+
+-   `port_name` — The name of the server port. [String](../../sql-reference/data-types/string.md#string).
```

Comment:
We need to add list of possible values here from original PR.

Reply:
> We need to add list of possible values here from original PR.

Ok.